api/docker-compose.yml

```diff
@@ -6,6 +6,9 @@ services:
     volumes:
       - "./cache:/cache:rw"
       - ./app:/usr/local/lib/python3.9/site-packages/app:ro
+      # Use these to run the app in development mode and update depedencies
+      # - ./pyproject.toml:/pyproject.toml
+      # - ./poetry.lock:/poetry.lock
```

Comment:
Could these be in `docker-compose.develop.yml` instead? Also, I think you'd need to mount the venv folder in the container for this to actually have any effect, no? Maybe we'd have to amend the venv location to live inside the project directory using the [right config option](https://python-poetry.org/docs/configuration/#virtualenvsin-project)?

Reply:
It seemed to have worked when I used it. But good call, I'll move this to .develop